src/virt.rs

```diff
@@ -732,6 +733,13 @@ impl VirtContext {
             MCause::MachineExternalInt => {
                 todo!("Virtualize machine external interrupt")
             }
+            MCause::LoadAddrMisaligned | MCause::StoreAddrMisaligned => {
+                self.emulate_jump_trap_handler()
+            }
+            MCause::InstrAddrMisaligned => {
+                todo!("Implement Instruction Address Misaligned")
+            }
```

Comment:
We shouldn't implement a default behavior in Miralis. If the policies does not handle it, the virtualized firmware should (and that is what happen with OpenSBI when we don't have the protect payload policy).